utils/azure_users.py

```diff
@@ -178,6 +178,41 @@ def update_role(self, user_id, role_id, is_grant):
 
         return self.to_azure_user_v2(response.json())
 
+    def _get_user(self, user_id):
+        endpoint = "{endpoint}/users/{user_id}?api-version=1.6".format(
+            endpoint=self.config.ENDPOINT, user_id=user_id
+        )
+        response = requests.get(endpoint, auth=BearerAuth(self.access_token))
+        assert 200 == response.status_code
+        return response.json()
+
+    def is_test_user(self, user_id):
+        response = self._get_user(user_id)
+        field_name, field_value = ROLE_FIELD_VALUES['test']
+        return field_name in response and field_value == response[field_name]
+
+    def _get_test_user_ids(self):
+        field_name, field_value = ROLE_FIELD_VALUES['test']
+        endpoint = "{endpoint}/users?api-version=1.6&$select=objectId,{field_name}&$filter={field_name} eq '{field_value}'".format(
+            endpoint=self.config.ENDPOINT,
+            field_name=field_name,
+            field_value=field_value,
+        )
+        response = requests.get(endpoint, auth=BearerAuth(self.access_token))
+        assert 200 == response.status_code
+        assert 'value' in response.json()
+        return response.json()['value']
+
+    def get_test_user_ids(self):
```

Comment:
Why do we have this "name duplicated" method

Reply:
We solved it. Explanation is here https://github.com/ioet/time-tracker-backend/pull/252#discussion_r563873423